src/addon/ESMX/Comps/ESMX_Data/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.19)
```

Comment:
Do you want to update this to 3.22 for consistency?

Reply:
I was thinking of keeping local minimum version requirements as where they really are... but thinking about it, because of the integration into ESMX that depends on 3.22 now, we might never know when 3.19 might really not be correct. So, I think it is best to bump them all to 3.22 now. Thanks, I will push that with the README.md update when I have it.